src/box/lua/load_cfg.lua

```diff
@@ -1012,6 +1012,7 @@ local box_cfg_guard_whitelist = {
     NULL = true;
     info = true;
     iproto = true;
+    priv = true;
```

Comment:
TBH, I don't get why it is needed. Can you explain, please?

Reply:
Because the `compat` module may be configured before `box.cfg` and the new option action uses `box.priv.X`. Anyway, it's safe to export `box.priv` because it stores only constants.

Reply:
Thanks, got it.